StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+API.swift

```diff
@@ -733,3 +744,21 @@ private func isEqual(_ lhs: STPPaymentIntentShippingDetails?, _ rhs: STPPaymentI
 
     return rhs == lhsConverted
 }
+
+private extension ConsumerPaymentDetails {
+    
+    func expectedPaymentMethodTypeForPassthroughMode(
+        _ elementsSession: STPElementsSession
+    ) -> String? {
+        switch type {
+        case .card:
+            return "card"
+        case .unparsable:
+            return nil
+        case .bankAccount:
+            let canAcceptACH = elementsSession.orderedPaymentMethodTypes.contains(.USBankAccount)
+            let isLinkCardBrand = elementsSession.linkSettings?.linkMode?.isPantherPayment ?? false
+            return isLinkCardBrand && !canAcceptACH ? "card" : "bank_account"
```

Comment:
This mimics behavior on the Web: If we’re in the Link card brand mode, but the merchant accepts ACH, then we want the payment method to be a full bank account.

cc @abagasra-stripe and @constantine-stripe

Reply:
Interesting, okay!